generators/client/templates/angular/webpack/webpack.common.js.ejs

```diff
@@ -40,6 +41,10 @@ module.exports = (options) => ({
     },
     module: {
         rules: [
+            {
+                test: /(?:\.ngfactory\.js|\.ngstyle\.js|\.ts)$/,
+                use: [ '@ngtools/webpack' ]
```

Comment:
brave move 😉 😜

Reply:
Haha yes ! Please try if you see any performance issues I'll rollback of course 😄

Reply:
Yeah sure 👍 
Did you find any good performance change? 
This also means that we will have to wait for angular-cli release cycle. `v6 - ivy` promises a lot let us see

Reply:
I'll do some tests to see the build time but it seemed pretty much the same...

Reply:
what matters is the time to compile when in hot reload mode for any changes to a file